client/src/modules/stock/lots/modals/duplicates.modal.html

```diff
@@ -0,0 +1,71 @@
+<form
+  name="ModalForm"
+  ng-submit="$ctrl.submit(ModalForm)"
+  data-modal="stock-lots-duplicates"
+  novalidate>
+
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.STOCK_LOTS</li>
+      <li class="title" translate>FORM.LABELS.DUPLICATE_LOTS</li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+    <div ng-if="$ctrl.lots.length === 1">
+      <span translate>LOTS.NO_LOTS_FOUND</span>
+    </div>
+    <div ng-if="$ctrl.lots.length > 1">
+      <span translate translate-values="{'N':$ctrl.lots.length - 1}">LOTS.FOUND_N_LOTS</span>
+      <table class="table table-condensed table-bordered">
+        <thead>
+          <th>Keep</th>
+          <th>Merge</th>
+          <th>Label</th>
+          <th>Quantity</th>
+          <th>Initial Quantity</th>
+          <th>Unit Price</th>
+          <th>Entry Date</th>
+          <th>Expiration Date</th>
+        </thead>
+        <tbody>
+          <tr ng-repeat="lot in $ctrl.lots track by lot.uuid"
+              ng-class="{ 'selected-lot' : lot.selected }">
+            <td class="center">
+              <input type="radio" name="selected" ng-if="lot.uuid === $ctrl.selectedLot.uuid"
```

Comment:
A similar idea can probably be applied to the `checkboxes`.

Reply:
Finally, let's add `style="cursor:pointer;"` to these radio buttons.  I'm surprised the browser does not default to it.  It will help our users know when they are over a "clickable area".

Reply:
Done.